src/OperationalInsights/OperationalInsights.Autorest/examples/Get-AzOperationalInsightsTable.md

```diff
@@ -0,0 +1,23 @@
+### Example 1: List tables for a given workspace name
```

Comment:
### Example 1: List tables for a given workspace name
```powershell
Get-AzOperationalInsightsTable -ResourceGroupName {RG-Name} -WorkspaceName {WS-Name}


Name                                         Id                                                                                                                                                                                                      RetentionInDays
----                                         --                                                                                                                                                                                                      ---------------
DatabricksTables                             /subscriptions/{SUB-id}/resourcegroups/{RG-Name}/providers/Microsoft.OperationalInsights/workspaces/{WS-Name}/tables/DatabricksTables                                          30
BlockchainProxyLog                           /subscriptions/{SUB-id}/resourcegroups/{RG-Name}/providers/Microsoft.OperationalInsights/workspaces/{WS-Name}/tables/BlockchainProxyLog                                        30
BlockchainApplicationLog                     /subscriptions/{SUB-id}/resourcegroups/{RG-Name}/providers/Microsoft.OperationalInsights/workspaces/{WS-Name}/tables/BlockchainApplicationLog                                  30
AADDomainServicesAccountLogon                /subscriptions/{SUB-id}/resourcegroups/{RG-Name}/providers/Microsoft.OperationalInsights/workspaces/{WS-Name}/tables/AADDomainServicesAccountLogon                             30
AADDomainServicesAccountManagement           /subscriptions/{SUB-id}/resourcegroups/{RG-Name}/providers/Microsoft.OperationalInsights/workspaces/{WS-Name}/tables/AADDomainServicesAccountManagement                        30
```

{{ Add example description here }}

### Example 2: Get a table by name
```powershell
Get-AzOperationalInsightsTable -ResourceGroupName {RG-Name} -WorkspaceName {WS-Name} -TableName {Table-Name}

Name  Id                                                                                                                                                               RetentionInDays
----  --                                                                                                                                                               ---------------
{Table-Name} /subscriptions/{SUB-id}/resourcegroups/{RG-Name}/providers/Microsoft.OperationalInsights/workspaces/{WS-Name}/tables/{Table-Name}              90
```

{{ Add example description here }}

Reply:
Hi @dabenhamMic , please check all your examples whether follow our format. Otherwise, our tests will fail in build

Reply:
For example content, it should be surrounded with \```

Reply:
hi @BethanyZhou , I fixed the issue you described but the build still fails.
will appreciate your help in resolving this iisue.

Reply:
Find some issues remaining.